src/core_ocean/Registry.xml

```diff
@@ -3091,6 +3091,22 @@
 			 description="Short wave flux at cell centers from coupler. Positive into the ocean."
 			 packages="activeTracersBulkRestoringPKG;ecosysTracersPKG"
 		/>
+		<var name="rainTemperatureFlux" type="real" dimensions="nCells Time" units="degC m s^{-1}"
+			 description="Heat flux associated with rain at cell centers sent to coupler. Positive into the ocean."
+		/>
+		<var name="evapTemperatureFlux" type="real" dimensions="nCells Time" units="degC m s^{-1}"
+			 description="Heat flux associated with Evaporation at cell centers sent to coupler. Positive into the ocean."
+		/>
+		<var name="seaIceFreshWaterTemperatureFlux" type="real" dimensions="nCells Time" units="degC m s^{-1}"
+			 description="Heat flux associated with sea ice melt water at cell centers sent to coupler. Positive into the ocean."
+		/>
+		<var name="icebergFreshWaterTemperatureFlux" type="real" dimensions="nCells Time" units="degC m s^{-1}"
+			 description="Heat flux associated with iceberg melt at cell centers sent to coupler. Positive into the ocean."
+		/>
+
+		<var name="totalFreshWaterTemperatureFlux" type="real" dimensions="nCells Time" units="degC m s^{-1}"
```

Comment:
@maltrud and @vanroekel it's unusual to have a temperature flux in degC m/s rather than a heat flux in W/m^2. I'm guessing these are the units received from the coupler, so we are conforming to units from the other components?

Reply:
Yes, essentially the same as my question above.

Reply:
In the driver, this field gets multiplied by rho_sw*cp_sw before being sent to the cpl -- so it does end up in W/m^2 there.

Reply:
my rationale is that the variables are called "*TemperatureFlux" so should have units of degC m/s.  it seems kind of silly to calculate these as a heat flux by multiplying by rho*cp, then dividing by it again before converting to a tendency.

Reply:
don't know why git decided to italicize part of my comment....

Reply:
@maltrud, okay, I guess that makes sense.

Your comment got italicized because you have two `*` symbols in there and you didn't use back-ticks to make them code.